lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -827,7 +827,7 @@ def request_headers(options)
       end
 
       def success_from(action, response, options)
-        if %w[RedirectShopper ChallengeShopper].include?(response.dig('resultCode')) && !options[:execute_threed] && !options[:threed_dynamic]
```

Comment:
decided to just remove  !options[:threed_dynamic] since !options[:execute_threed] is sufficient and there's a similar check in StripePI